src/main/java/com/github/platymemo/alaskanativecraft/AlaskaNativeCraftModels.java

```diff
@@ -0,0 +1,34 @@
+package com.github.platymemo.alaskanativecraft;
+
+import com.github.platymemo.alaskanativecraft.client.model.entity.DogsledEntityModel;
+import com.github.platymemo.alaskanativecraft.client.model.entity.HarpoonEntityModel;
+import com.github.platymemo.alaskanativecraft.client.model.entity.MooseEntityModel;
+import com.github.platymemo.alaskanativecraft.client.model.entity.PtarmiganEntityModel;
+import com.github.platymemo.alaskanativecraft.client.model.entity.SealEntityModel;
+import com.github.platymemo.alaskanativecraft.client.model.entity.feature.KuspukSkirtModel;
+import net.fabricmc.fabric.api.client.rendereregistry.v1.EntityModelLayerRegistry;
+import net.minecraft.client.render.entity.model.EntityModelLayer;
+import net.minecraft.util.Identifier;
+
+public class AlaskaNativeCraftModels {
+    public static final EntityModelLayer PTARMIGAN_MODEL = registerMain("ptarmigan");
+    public static final EntityModelLayer DOGSLED_MODEL = registerMain("dogsled");
+    public static final EntityModelLayer HARPOON_MODEL = registerMain("harpoon");
+    public static final EntityModelLayer MOOSE_MODEL = registerMain("moose");
+    public static final EntityModelLayer SEAL_MODEL = registerMain("seal");
+    public static final EntityModelLayer KUSPUK_SKIRT_MODEL = registerMain("kuspuk_skirt");
```

Comment:
```suggestion
    public static final EntityModelLayer PTARMIGAN = registerMain("ptarmigan");
    public static final EntityModelLayer DOGSLED = registerMain("dogsled");
    public static final EntityModelLayer HARPOON = registerMain("harpoon");
    public static final EntityModelLayer MOOSE = registerMain("moose");
    public static final EntityModelLayer SEAL = registerMain("seal");
    public static final EntityModelLayer KUSPUK_SKIRT = registerMain("kuspuk_skirt");
```
Should follow the same naming scheme that's used in `EntityModelLayers`. That is, just the entity name, no `_MODEL` suffix.

Reply:
Additional fixes in 32325d32e5b1957cfb67d94b1b2b630e70400700